ast.d

```diff
@@ -148,6 +150,30 @@ OperatorType parseOperatorType(string input)
     }
 }
 
+enum OperatorClass
+{
+    Math,
+    Relational,
+    Logical
+}
```

Comment:
For areas like this https://github.com/briansteffens/bshift/pull/6/files#diff-0a5963e354b6ffca029f3be2efa243a8R192